thorlcr/activities/keyedjoin/thkeyedjoinslave.cpp

```diff
@@ -1367,6 +1368,9 @@ class CKeyedJoinSlave : public CSlaveActivity, implements IJoinProcessor, implem
             keyManagers.reset(new std::vector<std::atomic<IKeyManager *>>(parts.size()));
             for (auto & k: *keyManagers)
                 k = nullptr;
+            contextLoggers.clear();
+            for (unsigned i=0; i<parts.size(); i++)
```

Comment:
trivial: perhaps this should call reset() in each (woudl need a CStatsContextLogger::reset added).

Reply:
The contextLoggers will have an empty array when it is first called.  This loop pushing back parts.size() items to the array is needed somewhere.  After adding the elements to the array, it's not worth calling reset for each item....should it be setting up the array elsewhere and calling reset on each item here?